widget-showcase/src/App.ts

```diff
@@ -253,30 +289,23 @@ export default class App extends AppBase<WidgetProperties> {
 					value: timepickerValue
 				}),
 			]),
-			v('div', { classes: this.classes(css.component) }, [
+			v('div', { classes: this.theme(css.component) }, [
 				w(Select, {
 					key: 'select',
+					getOptionDisabled: (option: any) => option.disabled,
+					getOptionLabel: (option: any) => option.label,
+					getOptionValue: (option: any) => option.value,
+					getOptionSelected: (option: any) => !!selectValue && option.value === selectValue,
+					label: 'Native select',
 					options: dataSmall,
 					value: selectValue,
 					theme: dojoTheme,
-					onChange: (option: OptionData) => {
+					onChange: (option: any) => {
```

Comment:
Use `SelectProperties` from `@dojo/widgets/select/Select`

Reply:
`Select` no longer exports an interface for option items (`SelectProperties` refers to the interface for the widget's own properties.)